packages/core/src/__tests__/model.test.ts

```diff
@@ -60,27 +60,27 @@ describe('matching', () => {
   })
 
   it('apply string value', () => {
-    const clone = model.createDocument(taskIds.class.Task, doc1)
+    const clone = model.newDoc(taskIds.class.Task, doc1._id, doc1)
```

Comment:
Why new method?

Reply:
It was two methods doing same., I've removed one, question is naming only.

Reply:
1. They are different: `createDocument` has optional `id` generating new one when not provided, `newDoc` always requires `id`
2. If they are same, why new name, is `createDocument` bad?

Reply:
I mean it seems `createDocument` used everywhere and has better signature, why removing it and replacing with `newDoc` everywhere? Or `newDoc` used more often?

Reply:
I think, it should be named `create` to match .

```typescript
/**
 * Define operations with object modifications.
 */
export interface OperationProtocol {
  /**
   * Perform creation of new document. Object ID will be automatically generated and assigned to object.
   */
  create<T extends Doc> (_class: Ref<Class<T>>, values: AnyLayout | Doc): Promise<T>
```

Reply:
Code is updated.

Reply:
So idea is to rename well established `createDocument` method for reason to match something in `OperationProtocol`? How many times `OperationProtocol.create` used in the codebase? Why not rename `OperationProtocol.create` to `createDocument` if you want some names to match?

Reply:
From other side, it seems semantic for this two methods are different. Will return as createDocument and think about name for OperationProtocol.create => store or something more clear.